src/current/_includes/v23.1/ui/transactions-filter.md

```diff
@@ -10,14 +10,24 @@
 
 The transaction fingerprints returned are determined by the selected **Search Criteria**:
 
-1. By default, the **Top** `100` transaction fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `CPU Time`, `Contention Time`, `Execution Count`, or `Transaction Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` transaction fingerprints **By** `Transaction Time` for the `Past Hour` are returned.
+
+1. To change the number of results returned, select `25`, `50`, `100`, or `500` from the **Top** dropdown. To return a larger number, select `More` and choose an option: `1000`, `5000`, `10000`.
+1. To change the sort column, from the **By** dropdown, select a commonly sorted column: `CPU Time`, `Contention Time`, `Execution Count`, or `Transaction Time`. To sort by other columns, select `More` from the dropdown and choose an option: `Max Memory`, `Network`, `Retries`, `Rows Processed`.
+{{site.data.alerts.callout_info}}
+The commonly sorted columns are indexed on the underlying system table to reduce page latency. The other columns listed under `More` are not indexed, so selecting one of them may increase the page loading time.
+{{site.data.alerts.end}}
 1. Select the [**Time Range**](#time-interval).
 1. Click **Apply**.
 
     The list of transactions that satisfy the search criteria is displayed. The results can be further [searched and filtered](#search-and-filter).
 
+The actual time range of the statistics is displayed at the top right of the results, for example, `Showing aggregated stats from 20:00 to 21:59 (UTC)`. If you select a preset time interval (not a custom time interval), be aware that the statistics displayed are not automatically refreshed and may become stale. To see the latest statistics, update the **Search Criteria** and apply.
+
+The displayed aggregated statistics may not include data from the last 10 minutes, which is the default for the [`sql.stats.flush.interval` cluster setting](#transaction-statistics). To reduce page loading time, the results use persisted statistics. For in-memory statistics, directly query the following in-memory table: `crdb_internal.cluster_transaction_statistics`.{% if page.cloud != true %} Alternatively, use the [**Active Executions** view](#active-executions-view).{% endif %}
+
 ### Time interval
 
 To view transaction fingerprints within a specific time interval, select a time interval from the selector at the top of the tab. The time interval field supports preset time intervals (1 Hour, 6 Hours, 1 Day, etc.) and custom time intervals. To select a custom time interval, click the time interval field and select **Custom time interval**. In the **Start (UTC)** and **End (UTC)** fields select or type a date and time.
```

Comment:
In the Search and filter section, can we call out that this searches and filters the results only AFTER applying the search criteria? 

This is the new interaction model.

Reply:
Added info callout.